examples/data-objects/todo/tests/todo.test.ts

```diff
@@ -4,8 +4,23 @@
  */
 
 import { globals } from "../jest.config";
+import { retryWithEventualValue } from "@fluidframework/test-utils";
 
 describe("ToDo", () => {
+    const getItemUrl = async (index: number) =>
```

Comment:
It feels like changes like that can and should go in as a separate PR, before you make any other changes.